### ISSUE TITLE
Buffer goes missing when using powershell

### ISSUE THREAD

**Issue by Tyriar** [microsoft/terminal#2410]
# Environment

```none
Windows build number: 10.0.18956.1000
Windows Terminal version (if applicable): 0.3.2171.0
Happens in VS Code too
```

# Steps to reproduce

1. Run or launch a powershell terminal
2. Press enter a bunch of times (`echo` also triggers the behavior)

# Expected behavior

Scrollback/scroll bar should appear

# Actual behavior

No scroll bar

![ps](https://user-images.githubusercontent.com/2193314/62886892-ec0da180-bcf0-11e9-87b4-c837e72dd6eb.gif)

**Comment by Tyriar** [comment]
Running `ls` will make a scroll bar appear.

**Comment by Tyriar** [comment]
This happens if you have scroll back as well, enter just redraws the entire screen or something, rather than actually pushing a row up. So the top viewport lines just go missing.

**Comment by DHowett-MSFT** [comment]
This is actually PowerShell/PSReadline#724. Shipping versions of PSReadline are emitting the destructive "scroll" VT sequence.

**Comment by binaryfunt** [comment]
Should I still be experiencing this bug? It only occurs when running Powershell in Terminal, and according to `Get-Module -ListAvailable`, I have
```
ModuleType Version    Name
---------- -------    ----
...
Script     2.0.0      PSReadline
```
Is there a fix/workaround?

**Comment by zadjii-msft** [comment]
Honestly I don't think I fully understand PSReadline versioning - I think from [this diff](https://github.com/powershell/psreadline/compare/v2.0.0-beta3...v2.0.0-beta4) it was fixed in PSReadline `v2.0.0-beta4`, but I'm not sure if that means it's in `2.0.0` or not.

@DHowett-MSFT should be able to clear things up for me.

I'm _pretty sure_ you shouldn't be seeing this anymore.

**Comment by Tyriar** [comment]
I was confused by this as well, in PS6 prerelease info doesn't show up but you can get it via:

```
Get-Module PSReadLine | % { $_.PrivateData.PSData }
```

Apparently PS7 shows prerelease info when you run `Get-Module PSReadLine`.
